docs/modules/ROOT/partials/ref-registry-pod-spec.adoc

```diff
@@ -0,0 +1,88 @@
+[#pod-spec]
+// Do not forget to update link text in related xref(s). Antora does not support automatic name if the link has a fragment.
+= Configuring {registry} deployment using `PodTemplate`.
+
+WARNING: This is a preview feature, which may evolve in the future releases.
+Make sure to test that your deployment works as expected before using this feature in production.
+Review the release notes of future releases for updates.
+
+`ApicurioRegistry` CRD now contains the `spec.deployment.podSpecPreview` field, that has the same structure as the field `spec.template.spec` in a Kubernetes deployment resource (the `PodSpec` struct in `PodTemplate`).
+With some restrictions, the {operator} will forward the data from this field to the corresponding field in {registry} deployment.
+This provides greater configuration flexibility, without the need for {operator} to natively support each use-case.
+The following table contains a list of subfields that will not be accepted by the operator, and will result in a configuration error:
+
+.Restrictions on the `podSpecPreview` subfields
+[%header,cols="4,3,4"]
+|===
+| `podSpecPreview` subfield | Status | Details
+
+| `affinity`
+| alternative exists
+| `spec.deployment.affinity`
+
+| `containers[*]`
+| restricted
+| Only a single container with *empty name field* must be defined
+
+| `containers[*].args`
+| reserved
+| -
+
+| `containers[*].command`
+| reserved
+| -
+
+| `containers[*].env`
+| alternative exists
+| `spec.configuration.env`
+
+ifdef::apicurio-registry[]
+| `containers[*].image`
+| alternative exists
+| `spec.deployment.image`
+endif::[]
+ifdef::service-registry[]
+| `containers[*].image`
+| reserved
+| -
+endif::[]
+
+| `containers[*].name`
+| reserved
+| -
+
+| `containers[*].ports`
+| reserved
+| -
+
+| `containers[*].workingDir`
+| reserved
+| -
+
+| `ephemeralContainers`
+| reserved
+| -
+
+| `imagePullSecrets`
+| alternative exists
+| `spec.deployment.imagePullSecrets`
+
+| `initContainers`
+| reserved
+| -
+
+| `tolerations`
+| alternative exists
+| `spec.deployment.tolerations`
+
+|===
+
+If you are setting a field in `podSpecPreview` or `podSpecPreview.containers[0]`, its value must be valid, as if you configured it in the {registry} deployment directly.
+The {operator} may still modify the values you provided, but it will not fix an invalid value or make sure a default value is present.
+For example, if you configure `spec.deployment.podSpecPreview.containers[0].resources.limits`, then the resource requests will be empty.
+
+NOTE: We are providing an option to configure `PodSpec` and not the full `PodTemplate` because {registry} pod annotations and labels can already be configured using the `spec.deployment.metadata.annotations` and `spec.deployment.metadata.labels` fields.
```

Comment:
```suggestion
NOTE: {operator} provides an option to configure `PodSpec` and not the full `PodTemplate` because {registry} pod annotations and labels can already be configured using the `spec.deployment.metadata.annotations` and `spec.deployment.metadata.labels` fields.
```